GitUI/UserControls/RevisionGrid/RevisionGridMenuCommands.cs

```diff
@@ -354,6 +354,13 @@ private IReadOnlyList<MenuCommand> CreateViewMenuCommands()
                     ExecuteAction = () => _revisionGrid.ToggleShowGitNotes(),
                     IsCheckedFunc = () => AppSettings.ShowGitNotes
                 },
+                new MenuCommand
+                {
+                    Name = "showMultiLineCommitMessagesToolStripMenuItem",
+                    Text = "Show multi-line commit messages",
```

Comment:
```suggestion
                    Text = "Show commit body",
```

Reply:
@pmiossec If this would also be ok for you, I will also change all the related variables, etc.

Reply:
> I will also change all the related variables, etc.

Yes, please. For `AppSettings`, I prefer to be even more specific (and to use Delphi-case for the new setting):
`"showmultilinecommitmessages"` --> `"ShowCommitBodyInRevisionGrid"`

Reply:
Should probably be clearer...
```suggestion
                    Text = "Show commit message body",
```